testdata/test-data-systematic.xml

```diff
@@ -294,7 +296,8 @@
 
     <region label="Region in image"
             id="region_1"
-            permissions="res-restricted">
+            permissions="res-restricted"
+            creationDate="1999-12-31T23:59:59.9999999+01:00">
```

Comment:
I think this is valid `xs:dateTime` but not `xs:dateTimeStamp`, could you maybe verify that? And speaking of, I think it would be good to add negative tests with invalid timestamps that should return an Error or a Warning.

Reply:
It is valid, because `xs:dateTimeStamp` is nothing else than a `xs:dateTime` with a mandatory timezone. My main problem, however, is that I don't know how I can make a reference to `xs:dateTimeStamp`, because it is defined in https://www.w3.org/TR/xmlschema11-2, and I don't know how I can "import" that.

As mentioned in previous comments, I'm unsure how to switch to XSD 1.1, but perhaps it boils down to "how can I import https://www.w3.org/TR/xmlschema11-2"?